textdb/textdb-sandbox/JinggangHelloWorld.java

```diff
@@ -1,6 +0,0 @@
-#include<iostream>
-public class JinggangHelloWorld{
-	public static void main(String[] args){
-		cout << "Hello World!" << endl;
```

Comment:
@chenlica By adding .DS_Store to the .gitignore file, it will ensure that Git does not add the file to  the repository ever, so we will never find it appearing here ever again.